ckanext/scheming/templates/scheming/form_snippets/license.html

```diff
@@ -9,7 +9,7 @@
 {%- if field.get('sorted_choices', true) -%}
     {%- set licenses = licenses|sort -%}
 {%- endif -%}
-{%- for license_desc, license_id in licenses -%}
+{%- for license_id, license_desc in licenses -%}
```

Comment:
is this just a separate bug fix?

Reply:
Yes,  we are no longer passing licenses as `c.attribute` with `dataset` Blueprint - but this change shouldn't be here, because it's not compatible with older CKAN versions. I'll remove it